src/Vehicle/VehicleBatteryFactGroup.cc

```diff
@@ -150,17 +213,21 @@ void VehicleBatteryFactGroup::_handleBatteryStatus(Vehicle* vehicle, mavlink_mes
         totalVoltage += cellVoltage;
     }
 
+    double current = batteryStatus.current_battery == -1 ? qQNaN() : static_cast<double>(batteryStatus.current_battery) / 100.0;
```

Comment:
why divided by 100?

Reply:
ah, it's centiAmpere lol https://mavlink.io/en/messages/common.html#BATTERY_STATUS